pkg/generators/active_set_tokens.go

```diff
@@ -58,10 +58,10 @@
 		})
 
 		sort.Slice(activeValidators, func(i, j int) bool {
-			return utils.StrToFloat64(activeValidators[i].DelegatorShares) > utils.StrToFloat64(activeValidators[j].DelegatorShares)
+			return activeValidators[i].DelegatorShares.GT(activeValidators[j].DelegatorShares)
```

Comment:
Done, can you check?

Reply:
It doesn't, but we do not need the sorting itself - we need to know the stake of the last validator, and it should be checked within these lines:
```
	gauge, ok := results[0].(*prometheus.GaugeVec)
	assert.True(t, ok)
	assert.InEpsilon(t, float64(2), testutil.ToFloat64(gauge.With(prometheus.Labels{
		"chain": "chain",
	})), 0.01)
```
This basically says that this metric should return 2 with this dataset (an active validator with stake 3, an active validator with stake 2, and an inactive validator with stake 1).

Reply:
> This indirect testing method is sufficient as long as it consistently reflects the intended behavior of the sorting logic, ensuring that the metric captures the correct stake value.

It is, as if the behaviour would change, the tests would break and I'll know about that.